contracts/DCAPermissionsManager/DCAPermissionsManager.sol

```diff
@@ -128,8 +128,11 @@ contract DCAPermissionsManager is ERC721, EIP712, Governable, IDCAPermissionMana
 
   /// @inheritdoc IDCAPermissionManager
   function modifyMany(PositionPermissions[] calldata _permissions) external {
-    for (uint256 i = 0; i < _permissions.length; i++) {
+    for (uint256 i = 0; i < _permissions.length; ) {
       modify(_permissions[i].tokenId, _permissions[i].permissionSets);
+      unchecked {
```

Comment:
I can't believe we have to do this manually and the compiler can't do it on its own. Truly a sad moment